src/style.css

```diff
@@ -0,0 +1,60 @@
+body {
+  margin: 0;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin: 0 4px;
```

Comment:
it's better use `:not(:last-child)` to set margin-right to all stars besides last one